deep-learning/_posts/2017-03-16-vggnet.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: review
+title: (VggNet) Very Deep Convolutional Networks for Large-Scale Image Recognition
```

Comment:
I think that "VggNet : Very Deep Convolutional..." would be more readable than using parentheses. If you use a colon you must surround the title with quotes.